.travis.yml

```diff
@@ -144,6 +144,8 @@ script:
     fi
   # different builds with CMake
   - if [ "$BUILD_MODE" == "cmake" ]; then
+      # Use early version of cmake
+      sudo -H $PIP install cmake==3.6.3
```

Comment:
need to make sure we pick cmake from /usr/local/bin before /usr/bin ? (if this is not the case already)